internal/clients/response.go

```diff
@@ -31,3 +31,8 @@ func IsResponseCodeInternalError(erro error) bool {
 		return strings.Contains(erro.Error(), fmt.Sprintf("[%d]", http.StatusInternalServerError))
 	}
 }
+
+type ErrorWithCode interface {
```

Comment:
Please consider documenting this since this is exported.

Reply:
fixed in https://github.com/hashicorp/terraform-provider-hcp/pull/826/commits/aa310a3a72deb8d82363513b328e2f666d76598a